js/src/hooks/useGoogleAdsAccount.js

```diff
@@ -46,6 +46,14 @@ const useGoogleAdsAccount = () => {
 				GOOGLE_ADS_ACCOUNT_STATUS.INCOMPLETE,
 			].includes( acc?.status );
 
+			const {
+				getAdsAccountStatus,
+				isResolving: isResolvingGoogleAdsAccountStatus,
+				hasFinishedResolution:
+					hasFinishedResolutionGoogleAdsAccountStatus,
+			} = select( STORE_KEY );
```

Comment:
Actually I've created a new hook to prevent any confusion. Also it'll prevent requests made to the `account-status` endpoint every time the `useGoogleAdsAccount` hook is called.